daemon/src/lib.rs

```diff
@@ -238,13 +239,15 @@ where
         min_quantity: Usd,
         max_quantity: Usd,
         fee_rate: Option<u32>,
+        funding_rate: FundingRate,
```

Comment:
Can we make this optional here? That would make this a backwards-compatible API change.

Reply:
sure!